### PR TITLE
[SonarCloud]: Create constant to avoid string duplication

### PR DESCRIPTION
**Pull request by iholder-redhat**
Signed-off-by: Itamar Holder <iholder@redhat.com>

**What this PR does / why we need it**:
The strings "Not Found" and "Bad Request" were
duplicated 11 times in api.go, now constants are defined to
avoid duplication.

Discovered by SonarCloud:
tinyurl.com/46q62jcl

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-api/api.go [hunk 1]
```diff
@@ -74,6 +74,9 @@ const (
 	defaultTlsKeyFilePath      = "/etc/virt-api/certificates/tls.key"
 	defaultHandlerCertFilePath = "/etc/virt-handler/clientcertificates/tls.crt"
 	defaultHandlerKeyFilePath  = "/etc/virt-handler/clientcertificates/tls.key"
+
+	notFound   = "Not Found"
+	badRequest = "Bad Request"
 )
```
Comment (ezrasilvera) [inline pkg/virt-api/api.go:80]: Maybe somehow indicate this is a string/message/label it's not so clear when it's used (`Returns(http.StatusNotFound, notFound, "")` )  ?

Comment (EdDev) [reply pkg/virt-api/api.go:80]: +1

It should tell what it is. E.g: `httpStatusBadRequestMessage`

**Review by EdDev — CHANGES_REQUESTED** [review]
> tinyurl.com/46q62jcl

Please do not use such links as they are not permanent and will eventually break.
The commit is here to stay forever, and these kind of links will usually be cleaned after a few months.

**Review by EdDev — APPROVED** [review]
Thank you!

**Pull request merged**